Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.8.16'
```

Comment:
I think we are already on `1.9.12`.

Reply:
Updated it to 1.9.13 since this is the last commit